pkg/kubecost/summaryallocation.go

```diff
@@ -39,6 +39,7 @@ type SummaryAllocation struct {
 	SharedCost             float64               `json:"sharedCost"`
 	ExternalCost           float64               `json:"externalCost"`
 	Share                  bool                  `json:"-"`
+	SharedCostBreakdown    SharedCostBreakdowns  `json:"sharedCostBreakdown"`
```

Comment:
Nick, I assume you are including this because the allocations object isn't built by this function, and therefore have no way to return this to the FE? 

If so, my vote would be to wait until https://kubecost.atlassian.net/browse/SELFHOST-284 gets implemented, which would address this issue. I am not crazy about changing API behavior just to work around our own planning snafu that I introduced